src/CacheClient.cpp

```diff
@@ -0,0 +1,198 @@
+#include "CacheClient.hpp"
+#include "Hash.hpp"
+#include "DagData.hpp"
+#include "RunAction.hpp"
+#include "NodeResultPrinting.hpp"
+#include "StatCache.hpp"
+#include "Profiler.hpp"
+#include "Exec.hpp"
+#include "BuildQueue.hpp"
+#include "MakeDirectories.hpp"
+
+const char* kENV_REAPI_CACHE_CLIENT = "REAPI_CACHE_CLIENT";
+const char* kENV_CACHE_SERVER_ADDRESS = "CACHE_SERVER_ADDRESS";
```

Comment:
Should we add a comment describing the format here? (I will also check that reapi-cache-client gives a useful error message if the address is malformed.)

Reply:
`2020/08/31 10:00:31 FindMissingBlobs: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp: address servernamehere: missing port in address"`

Reply:
Added some comments describing all of these.